### PR TITLE
Added new POI's.

### PR DESCRIPTION
**Pull request by ThoughtSynapsis**
Addition of new POI's for new Submap Generation.

### CODE REVIEW

**Comment by Anewbe** [comment]
maps/submaps/surface_submaps/forest.dm:16:error: unable to open "Snowrock1.dmm".

File: maps/submaps/cave_submaps/cave.dm [hunk 1]
```diff
@@ -5,6 +5,8 @@
 #include "deadBeacon.dmm"
 #include "prepper1.dmm"
 #include "quarantineshuttle.dmm"
+#include "Mineshaft1"
+#include "Scave1"
```
Comment (Atermonera) [inline maps/submaps/cave_submaps/cave.dm]: These aren't map files

File: maps/submaps/cave_submaps/cave.dm [hunk 2]
```diff
@@ -5,6 +5,8 @@
 #include "deadBeacon.dmm"
 #include "prepper1.dmm"
 #include "quarantineshuttle.dmm"
+#include "Mineshaft1"
```
Comment (Neerti) [inline maps/submaps/cave_submaps/cave.dm]: Put ``.dmm`` at the end for this and the one below.

File: maps/submaps/surface_submaps/forest.dm [hunk 3]
```diff
@@ -20,4 +30,49 @@
...
-	mappath = 'maps/submaps/surface_submaps/spider1.dmm'
\ No newline at end of file
+	mappath = 'maps/submaps/surface_submaps/spider1.dmm'
+
+/datum/map_template/surface/Flake
+	name = "Forest Lake"
+	desc = "A serene lake sitting amidst the surface."
+	mappath = 'maps/submaps/surface_submaps/Flake1.dmm'
+
+/datum/map_template/surfance/Mcamp1
```
Comment (Atermonera) [inline maps/submaps/surface_submaps/forest.dm]: Surfance isn't a word, afaik

**Pull request merged**